SECURITY.md

```diff
@@ -7,9 +7,6 @@ The list of supported versions can be found
 
```

Comment:
I can't comment on the above line, but it should be:
```suggestion
[here](https://goteleport.com/download/).
```

Reply:
Addressed in a follow-up commit to update all the old download URLs.